ptx/S_12_3_ParamLineIntegrals.ptx

```diff
@@ -2,427 +2,264 @@
-      calculating a line integral of a vector field along some line segments
-      and will use this as inspiration to see how treating oriented curves
-      as vector-valued functions can allow us to quickly turn a line
-      integral of a vector field into an integral of a single variable.
+      We begin this section by taking a look at how we can calculate a line integral of a vector field along some line segments and use this calculation as inspiration to see how treating oriented curves as vector-valued functions will allow us to quickly turn a line integral of a vector field into a single variable integral.
     </p>
+  </introduction>
 
-    <exploration xml:id="PA_12_3">
+  <exploration xml:id="PA_12_3">
+    <introduction>
       <p>
-        Let <m>\vF=\langle xy,y^2\rangle</m>,
-        let <m>C_1</m> be the line segment from
-        from <m>(1,1)</m> to <m>(4,1)</m>,
-        let <m>C_2</m> be the line segment from <m>(4,1)</m> to
-        <m>(4,3)</m>,
-        and let <m>C_3</m> be the line segment from <m>(1,1)</m> to
-        <m>(4,3)</m>.
-        Also let <m>C = C_1 + C_2</m>.
-        This vector field and the curves
-        are shown in <xref ref="F_12_3_field-segments">Figure</xref>.
+        Let <m>\vF=\langle xy,y^2\rangle</m>, let <m>C_1</m> be the line segment from from <m>(1,1)</m> to <m>(4,1)</m>, let <m>C_2</m> be the line segment from <m>(4,1)</m> to <m>(4,3)</m>, and let <m>C_3</m> be the line segment from <m>(1,1)</m> to <m>(4,3)</m>. Also let <m>C = C_1 + C_2</m>. This vector field and the curves are shown in <xref ref="F_12_3_field-segments">Figure</xref>.
       </p>
-
       <figure xml:id="F_12_3_field-segments">
         <caption>A vector field <m>\vF</m> and three oriented curves.</caption>
         <image source="images/fig_12_3_preview_field_segments_rev"/>
       </figure>
-
-      <ol>
-        <li>
+    </introduction>
+        <task xml:id="PA_12_3_task1">
           <p>
-            Every point along <m>C_1</m> has <m>y=1</m>.
-            Therefore, along <m>C_1</m>, the
-            vector field <m>\vF</m> can be viewed purely as a function of <m>x</m>.
-            In
-            particular, along <m>C_1</m>,
-            we have <m>\vF(x,1) = \langle
-            x,1\rangle</m>.
-            Since every point along <m>C_2</m> has the same <m>x</m>-value, what
-            (in terms
-            of <m>y</m> only)
-            is <m>\vF</m> along <m>C_2</m>?
+            Every point along <m>C_1</m> has <m>y=1</m>. Therefore, along <m>C_1</m>, the vector field <m>\vF</m> can be viewed purely as a function of <m>x</m>. In particular, along <m>C_1</m>, we have <m>\vF(x,1) = \langle x,1\rangle</m>. Since every point along <m>C_2</m> has the same <m>x</m>-value, what (in terms of <m>y</m> only) is <m>\vF</m> along <m>C_2</m>?
           </p>
-        </li>
-
-        <li>
+          <answer> <p> <m>\vF=\langle 4y,y^2\rangle </m> along <m>C_2</m>. </p> </answer>
```

Comment:
Even if the `solution` winds up being identical to the `answer`, we need to have them both. Typical pattern will be to send `answer` to backmatter and use `solution` to produce a solutions manual. It *may* be possible to write some XSLT that puts the `answer` into the solutions manual when there is no `solution`, but it seems that the current preferred practice is to use them both.